pattern-library/partials/_pl-head.html

```diff
@@ -11,5 +11,6 @@
     <!--<link href='http://fonts.googleapis.com/css?family=Lato:400,900,400italic,900italic' rel='stylesheet' type='text/css'>-->
     <link href='../../fonts/Lato/css/fonts.css' rel='stylesheet' type='text/css'>
     <link id="stylesheet" href="../../css/style.css" rel="stylesheet" type="text/css" />
+    <link id="stylesheet" href="../../css/rtl-style.css" rel="stylesheet" type="text/css" />
```

Comment:
So we can now load both stylesheets at once. Which means we can lazy load them in the client... but don't have to worry so much about disabling one when using the other.

Ideally we might eventually move some bits to a common stylesheet, or just a `loading` stylesheet so we can show a spinner while we figure out if the site should be RTL or LTR for the current user

Reply:
this is a good idea. I was thinking about loading both at the same time, but was unsure of the best approach. But I love the idea of loading stylesheet to give us time to determine RTL or LTR.